### PR TITLE
Fix authentication

### PR DESCRIPTION
**Pull request by atstojanov**
### Describe the background of your pull request

What does your pull request do? Does it solve a bug (which one?), add a feature?

### Additional context

Add any other context

### Governance 

- [ ] Documentation is added
- [ ] Test cases are added or updated
- [ ] I've read the contributing document https://github.com/cherrytwist/.github/blob/master/CONTRIBUTING.md
- [ ] I've read and understand the Code of Conduct https://github.com/cherrytwist/.github/blob/master/CODE_OF_CONDUCT.md
- [ ] I understand that any contributions or suggestions I made may make it into the actual code. I've read the License 
     https://github.com/cherrytwist/Coordination/blob/master/LICENSE 
     and the contributor license agreement: tba

### CODE REVIEW

**Review by OlegSubik — DISMISSED** [review]
please remove all redundant comments, consoles and a dubugger

File: src/hooks/useGraphQLClient.ts [hunk 1]
```diff
@@ -11,46 +11,107 @@ import { ApolloClient } from '@apollo/client/core/ApolloClient';
...
+  // Preserve the token and status from the reduxStore to be used inside the memoized apollo client
+  const status = useRef<AuthStatus>('unauthenticated');
+  const token = useRef<string | undefined>();
+  token.current = _token;
+  status.current = _status;
+
+  const pendingRequests = useRef<((token?: string) => void)[]>([]);
+  const isRefreshing = useRef(false);
+  const counter = useRef(0);
+  const refreshCounter = useRef(0);
```
Comment (OlegSubik) [inline src/hooks/useGraphQLClient.ts:39]: i dont see the usage of it. seeing only the increment on line 47

Comment (atstojanov) [reply src/hooks/useGraphQLClient.ts:39]: Oh yes, they were just for development purpose.

**Review by techsmyth — APPROVED** [review]
Think these were the only changes needed from Oleg's review, so bringing through...

**Pull request merged**